### PR TITLE
Update codeowners for rich-text components

### PR DESCRIPTION
**Pull request by m-akinc**
# Pull Request

## 🤨 Rationale

vikisekarNI is apparently no longer a valid owner, so replacing with vivinkrishna-ni.

## 👩‍💻 Implementation

See above.

## 🧪 Testing

N/A

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -16,8 +16,8 @@
 /packages/nimble-components @rajsite @jattasNI
 /packages/nimble-components/.storybook @rajsite @jattasNI @fredvisser
 /packages/nimble-components/src/wafer-map @rajsite @jattasNI @DStavilaNI @zszilagy
-/packages/nimble-components/src/rich-text-viewer @rajsite @jattasNI @vikisekarNI
-/packages/nimble-components/src/rich-text-editor @rajsite @jattasNI @vikisekarNI
+/packages/nimble-components/src/rich-text-viewer @rajsite @jattasNI @vivinkrishna-ni
```
Comment (rajsite) [inline .github/CODEOWNERS:19]: Added @vivinkrishna-ni to PR to approve being CODEOWNER for rich-text components (or let us know who should be owner).

Comment (vivinkrishna-ni) [reply .github/CODEOWNERS:19]: Sure, I can be a CODEOWNER for the changes related to rich test components.

I think the folder path needs to be updated to `src/rich-text` and `src/rich-text-mention` in this file.

**Review by vivinkrishna-ni — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -16,8 +16,8 @@
...
+/packages/nimble-components/src/rich-text-editor @rajsite @jattasNI @vivinkrishna-ni
```
Comment (rajsite) [inline .github/CODEOWNERS:20]:
```suggestion
/packages/nimble-components/src/rich-text @rajsite @jattasNI @vivinkrishna-ni
/packages/nimble-components/src/rich-text-mention @rajsite @jattasNI @vivinkrishna-ni
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (rajsite) [reply .github/CODEOWNERS:19]: Sounds good! Thanks!

**Review by rajsite — APPROVED** [review]

**Pull request merged**